src/components/Card/Card.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import "./Card.scss";
+import iconArrow from "../../assets/arrow-upper-right-black.svg";
+import "../../styles/_reset.scss";
+
+const Card = ({ img, dateOrTime, title, cardInfo, links }) => {
+
```

Comment:
typeof links === "string" ? links=[links] : links; <- adding this line will fix the problem with card not displaying if String is passed instead of the list